front/src/components/LoginPanel/comp/RegisterForm.js

```diff
@@ -96,6 +105,7 @@ const RegisterForm = (props) => {
                     className="form-control register-input"
                     value={password1}
                     onChange={handlePassword1Change}
+                    autoComplete="on"
```

Comment:
I am not an expert, but autocomplete have more options than just on and of. It probably should be:
```suggestion
                    autocomplete="new-password"
```